ResearchUXFactoryTests/ORKFormStep+Result.swift

```diff
@@ -267,6 +267,21 @@ extension ORKHeightAnswerFormat : SBAQuestionResultMapping {
     }
 }
 
+// TODO: syoung 08/01/31 Uncomment with transition to RK2
```

Comment:
commenting from twelve years in the future?

Reply:
Yes. ;)